src/uct/ib/base/ib_iface.c

```diff
@@ -148,9 +148,10 @@ ucs_config_field_t uct_ib_iface_config_table[] = {
    "Force interface to use global routing.",
    ucs_offsetof(uct_ib_iface_config_t, is_global), UCS_CONFIG_TYPE_BOOL},
 
-  {"SL", "0",
-   "IB Service Level / RoCEv2 Ethernet Priority.\n",
-   ucs_offsetof(uct_ib_iface_config_t, sl), UCS_CONFIG_TYPE_UINT},
+  {"SL", "auto",
+   "InfiniBand: Service level. 'auto' will select a value matching UCX_IB_AR configuration.\n"
```

Comment:
There should not be such var, AR is relevant for MLX5 tls only

Reply:
we have such a variable (since ), but it has effect only for MLX5 TLs

Reply:
I mean `UCX_IB_AR` you defined it MLX5 only.

Reply:
> I mean `UCX_IB_AR` you defined it MLX5 only.

yes, I see
but it could be used as `UCX_IB_AR`